spreadsheet/formula/fndatetime.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased by contacting sales@baliance.com.
```

Comment:
Please ensure that this email address is NOT used anywhere.  We are sales@foxyutils.com

Reply:
support@unidoc.io ?

Reply:
replaced for support@unidoc.io